packages/ui-range-input/src/RangeInput/index.tsx

```diff
@@ -169,12 +166,18 @@ class RangeInput extends Component<RangeInputProps> {
             htmlFor={this.id}
             css={this.props.styles?.rangeInputInputValue}
           >
-            {/* @ts-expect-error ts-migrate(2722) FIXME: Cannot invoke an object which is possibly 'undefin... Remove this comment to see the full error message */}
-            {this.props.formatValue(this.value)}
+            {/**
+             * Value can be undefined when no value or defaultValue is given,
+             * and then it renders an empty ContextView
+             * TODO: Ask a designer what the intended behaviour is
+             */}
+            {this.props.formatValue!(this.value, this.props.max)}
```

Comment:
Opened a ticket about this: INSTUI-3356

Reply:
Little more explanation: the VRT made me notice this bug after typing it correctly, but I wasn't sure what the best solution is to fix it, so I reverted this part, added a comment and made a ticket.